CodeMaid.config

```diff
@@ -16,6 +16,12 @@
       <setting name="Reorganizing_RunAtStartOfCleanup" serializeAs="String">
         <value>False</value>
       </setting>
+      <setting name="Feature_CleanupActiveCode" serializeAs="String">
```

Comment:
Because I see you force auto cleanup on file save for this repo, this feature requires `Feature_CleanupActiveCode` and `Feature_SettingCleanupOnSave`  features on.

Reply:
Ok I see, so you're trying to ensure those features are present in case someone normally has them disabled in their user-level settings.

Reply:
Yes.